docs/cookbook/deployment/platform-sh.rst

```diff
@@ -19,189 +13,108 @@ In this guide you will find sufficient instructions to have your application up
-            php bin/console --env=prod --no-debug --ansi sylius:theme:assets:install
-            yarn install
-            GULP_ENV=prod yarn build
-        deploy: |
-            rm -rf var/cache/*
-            php bin/console --env=prod doctrine:migrations:migrate --no-interaction
-
-The above configuration includes tuned cache expiration headers for static files. The cache lifetimes can be adjusted for your site if desired.
-
-* Add ``.platform/routes.yaml`` file:
-
-.. code-block:: yaml
-
-    # .platform/routes.yaml
-    "http://{default}/":
-        type: upstream
-        upstream: "app:http"
-
-    "http://www.{default}/":
-        type: redirect
-        to: "http://{default}/"
-
-* Add ``.platform/services.yaml`` file:
-
-This file will load ``mysql`` and ``redis`` on your Platform.sh server.
-
-.. code-block:: yaml
-
-    # .platform/services.yaml
-    mysql:
-        type: mysql
-        disk: 1024
-
-    redis:
-        type: redis:3.0
-
-* Configure the access to the database:
-
-In the ``app/config/parameters_platform.php`` file, put the following code:
-
-.. code-block:: php
-
-    // app/config/parameters_platform.php
-    <?php
-
-    $relationships = getenv("PLATFORM_RELATIONSHIPS");
-
-    if (!$relationships) {
-        return;
-    }
-
-    $relationships = json_decode(base64_decode($relationships), true);
-
-    foreach ($relationships['database'] as $endpoint) {
-        if (empty($endpoint['query']['is_master'])) {
-            continue;
-        }
-
-        $container->setParameter('database_driver', 'pdo_' . $endpoint['scheme']);
-        $container->setParameter('database_host', $endpoint['host']);
-        $container->setParameter('database_port', $endpoint['port']);
-        $container->setParameter('database_name', $endpoint['path']);
-        $container->setParameter('database_user', $endpoint['username']);
-        $container->setParameter('database_password', $endpoint['password']);
-        $container->setParameter('database_path', '');
-    }
-    foreach ($relationships['redis'] as $endpoint) {
-        $container->setParameter('redis_dsn', 'redis://'.$endpoint['host'].':'.$endpoint['port']);
-    }
-
-    ini_set('session.save_path', '/tmp/sessions');
-
-    if (getenv('PLATFORM_PROJECT_ENTROPY')) {
-        $container->setParameter('secret', getenv('PLATFORM_PROJECT_ENTROPY'));
-    }
-
-Remember to have it imported in the config:
-
-.. code-block:: yaml
-
-    # app/config/config.yml
-    imports:
-        # - { resource: parameters.yml } <- Has to be placed before our new file
-        - { resource: parameters_platform.php }
+    # The configuration of app when it is exposed to the web.
+    web:
+        locations:
+            "/":
+                # The public directory of the app, relative to its root.
+                root: "public"
+                # The front-controller script to send non-static requests to.
+                passthru: "/index.php"
```

Comment:
I'm wondering if we shouldn't keep caching config for default installation https://github.com/Sylius/Sylius/pull/10916/files#diff-5366644cbd212c7b2962e464bc992943L75